src/pages/index.js

```diff
@@ -194,11 +217,97 @@ const popupWithAddForm = new PopupWithForm(
   "#add-card-modal",
   handleAddCardSubmit
 );
+
 popupWithEditForm.setEventListeners();
 popupWithAddForm.setEventListeners();
 
+const cardSection = new Section(
+  {
+    items: initialCards,
+    renderer: (item) => {
+      const card = createCard(item);
+      cardSection.addItem(card);
+      ``` 1`;
+    },
+  },
+  ".cards__list"
+);
+
+const userData = new UserInfo(profileName, profileDescription, avatar);
+
+api
+  .getInitialCards()
+  .then((cardData) => {
+    const cardSection = new Section(
+      {
+        items: cardData,
+        renderer: (item) => {
+          const card = createCard(item);
+          cardSection.addItem(card);
+        },
+      },
+      ".cards__list"
+    );
+    const cardList = cardSection.renderItems();
+  })
+
+  .catch((err) => {
+    console.error(err); // log the error to the console
+  });
+
+api
+  .getUserInfo()
+  .then((data) => {
+    userData.setUserInfo(data.name, data.about);
+    userData.setUserAvatar(data.avatar);
+  })
+  .catch((err) => {
+    console.error(err); // log the error to the console
+  });
+
+const deleteCardPopUp = new PopUpWithConfirm(
+  "#delete-card-modal",
+  deleteCardSubmit
+);
+
+avatarUpdateButton.addEventListener("click", () => {
+  avatarCardPopUp.open();
+});
 
-/* profileEditForm.addEventListener("submit", handleProfileEditSubmit);
-addCardForm.addEventListener("submit", handleAddCardSubmit); */
-/* initialCards.forEach((cardData) (cardData, cardsWrap)); */
+function handleLikeClick(card) {
+  console.log(card._isLiked);
+  if (card._isLiked) {
+    api.dislikeCard(card._id).then(() => {
+      console.log(card._id);
+      card.handleLikeIcon();
+      card._isLiked = false;
+    });
+  }
+  if (!card._isLiked) {
+    api.likeCard(card._id).then(() => {
+      console.log(card._id);
+      card.handleLikeIcon();
+      card._isLiked = true;
```

Comment:
`_isLiked`, `_id`


Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.